test.js

```diff
@@ -135,3 +135,26 @@ test('translate via custom tld', async t => {
     t.is(res.from.text.value, '');
     t.false(res.from.text.didYouMean);
 });
+
+test('translate long text over 2083 chars using any language', async t => {
+    const text = `
```

Comment:
Let's generate long text programmatically, e.g. `'hi '.repeat(1500)`.

Reply:
Would that allow Google Translate to actually translate it, though? That's the question I have with that.

Reply:
I've made the change you suggested. Check and see if it translates with it as it is.